packages/simulator/web/routes.js

```diff
@@ -190,12 +191,9 @@ function register(io) {
         }
       })
 
-    experiment.buses
```

Comment:
Seems to be a duplicate from 
https://github.com/PredictiveMovement/digital-twin/pull/137/files#diff-825d8a145be1473656e5f100ade6eacea16a05c7a745c5e7d2cb7832a2989223L171